addons/l10n_cl/models/res_partner.py

```diff
@@ -37,8 +37,11 @@ def _format_vat_cl(self, values):
         partner_country_is_chile = country.code == "CL" or identification_type.country_id.code == "CL"
         if partner_country_is_chile and \
                 values.get('l10n_latam_identification_type_id') in identification_types and values.get('vat'):
-            return stdnum.util.get_cc_module('cl', 'vat').format(values['vat']).replace('.', '').replace(
-                'CL', '').upper()
+            try:
+                return stdnum.util.get_cc_module('cl', 'vat').format(values['vat']).replace('.', '').replace(
+                    'CL', '').upper()
+            except IndexError:
+                return values['vat']
```

Comment:
Sorry for the long wait! How about doing this?
```suggestion
        if partner_country_is_chile and \
                values.get('l10n_latam_identification_type_id') in identification_types and values.get('vat') and\
                stdnum.util.get_cc_module('cl', 'vat').is_valid(values['vat']):
            return stdnum.util.get_cc_module('cl', 'vat').format(values['vat']).replace('.', '').replace(
                    'CL', '').upper()
```
see the is_valid function in python-stdnum -> https://github.com/arthurdejong/python-stdnum/blob/master/stdnum/cl/rut.py

Reply:
Hello @antoine162 
Changes are done!
It is working properly.
Please review it.
Thanks!